src/components/screen-header/ScreenHeader.tsx

```diff
@@ -168,7 +168,8 @@ const BaseHeader = ({
 const useHeaderStyle = StyleSheet.createThemeHook((theme) => ({
   container: {
     paddingHorizontal: theme.spacings.medium,
-    paddingVertical: theme.spacings.large,
+    paddingTop: theme.spacings.large,
+    paddingBottom: theme.spacings.medium,
```

Comment:
Did you check other pages without the global message box, if they are still the same?

Reply:
Done now, fixed in the second commit